common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerRequest.java

```diff
@@ -31,14 +31,23 @@
 
 import java.io.Serializable;
 
+import lombok.Builder;
+import lombok.Getter;
+import lombok.experimental.Accessors;
+
 /**
  * Represents the broker request
  */
+@Builder
+@Accessors(prefix = "m")
+@Getter
 public class BrokerRequest implements Serializable {
 
-    private static final long serialVersionUID = 542024304389420837L;
+    private static final long serialVersionUID = -543392127065130474L;
+    // private static final long serialVersionUID = 542024304389420837L;
```

Comment:
I don't assume we need to keep this?

Reply:
Before I push this change, I'll remove it, for sure.